### ISSUE TITLE
`uv tool list` might have a `--plain` flag to only list names?

### ISSUE THREAD

**Issue by baggiponte** [astral-sh/uv#5183]
This might be a request of a solution for the wrong problem. Since I cannot install (or upgrade) multiple tools with `uv tool`, I thought of using a bit of a hack:

```bash
for tool in $(uv tool list); do
    uv tool uninstall $tool && uv tool install $tool
done
```

However, `uv tool list` doesn't simply print all tools, but also their entrypoints. Maybe a `--plain` or `--tools-only` or `--no-entrypoints` flag might do?

I remember opening a similar feature request for `rye` to accept multiple packages at once, but Armin correctly pointed out the executable would not be able to handle the options/flags.

**Comment by zanieb** [comment]
Note I think you can also do `uv tool install <name> --upgrade` and skip the `uninstall`?

**Comment by baggiponte** [comment]
> Note I think you can also do `uv tool install <name> --upgrade` and skip the `uninstall`?

Totally forgot about that. Thanks 😬

Feel free to close the issue whenever you want! And thanks again.

**Comment by charliermarsh** [comment]
I still think `tool list --plain` or similar would need to include versions though.

**Comment by zanieb** [comment]
Yeah idk, `--no-show-executables` and `--no-show-versions?`. Eventually it just makes sense to create a JSON output format.

**Comment by saaketp** [comment]
For comparison, [`pipx list`](https://pipx.pypa.io/latest/reference/cli/#pipx-list) has `--short` to show only packages and versions without the entrypoints, and `--json` for json output.

**Comment by drewlwhitney** [comment]
I would be interested in having an option like this so I can export a list of packages installed with `uv tool` and then install them on another system (i.e. backing them up in my dotfiles)
